x-pack/plugins/security/public/management/api_keys/api_keys_grid/empty_prompt/empty_prompt.tsx

```diff
@@ -5,72 +5,70 @@
  */
 
 import React, { Fragment } from 'react';
-import { ApplicationStart } from 'kibana/public';
 
 import { EuiEmptyPrompt, EuiButton, EuiLink } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
-import { DocumentationLinksService } from '../../documentation_links';
+import { useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
 
 interface Props {
   isAdmin: boolean;
-  docLinks: DocumentationLinksService;
-  navigateToApp: ApplicationStart['navigateToApp'];
 }
 
-export const EmptyPrompt: React.FunctionComponent<Props> = ({
-  isAdmin,
-  docLinks,
-  navigateToApp,
-}) => (
-  <EuiEmptyPrompt
-    iconType="managementApp"
-    title={
-      <h1 data-test-subj="noApiKeysHeader">
-        {isAdmin ? (
+export const EmptyPrompt: React.FunctionComponent<Props> = ({ isAdmin }) => {
+  const { services } = useKibana();
+  const application = services.application!;
+  const docLinks = services.docLinks!;
+  return (
+    <EuiEmptyPrompt
+      iconType="managementApp"
+      title={
+        <h1 data-test-subj="noApiKeysHeader">
+          {isAdmin ? (
+            <FormattedMessage
+              id="xpack.security.management.apiKeys.table.emptyPromptAdminTitle"
+              defaultMessage="No API keys"
+            />
+          ) : (
+            <FormattedMessage
+              id="xpack.security.management.apiKeys.table.emptyPromptNonAdminTitle"
+              defaultMessage="You don't have any API keys"
+            />
+          )}
+        </h1>
+      }
+      body={
+        <Fragment>
+          <p>
+            <FormattedMessage
+              id="xpack.security.management.apiKeys.table.emptyPromptDescription"
+              defaultMessage="You can create an {link} from Console."
+              values={{
+                link: (
+                  <EuiLink href={`${docLinks.links.apis.createApiKey}`} target="_blank">
```

Comment:
nit: Shouldn't the doc link be a string in itself? I don't think we need to wrap it in template literals

Reply:
It should, just kept it as it was before - will fix.